packages/framework/aqueduct/src/container-runtime-factories/containerRuntimeFactoryWithDefaultDataStore.ts

```diff
@@ -26,8 +26,8 @@ export class ContainerRuntimeFactoryWithDefaultDataStore extends BaseContainerRu
     public static readonly defaultDataStoreId = defaultDataStoreId;
 
     constructor(
-        protected readonly defaultFactory: IFluidDataStoreFactory,
-        registryEntries: NamedFluidDataStoreRegistryEntries,
+        private readonly defaultFactory: IFluidDataStoreFactory,
+        registryEntries: IFluidDataStoreRegistry,
```

Comment:
Should this be IFluidDataStoreRegistry or IProvideFluidDataStoreRegistry?

Reply:
I'm not confident it's better, but I've changed it - please take a look at last commit.